cmd/root/bundle_test.go

```diff
@@ -51,12 +50,11 @@ workspace:
 	err := os.WriteFile(filepath.Join(rootPath, "databricks.yml"), []byte(contents), 0o644)
 	require.NoError(t, err)
 
-	b, diags := MustConfigureBundle(cmd)
-	require.NoError(t, diags.Error())
-	return b
+	_, diags := MustConfigureBundle(cmd)
+	return diags.Error()
```

Comment:
Q: the bundle object is never used here, and only the resulting SDK config?

Reply:
Yeah, the idea is to get the config object, to pass it to the sub-process we spawn for telemetry.